data-prepper-core/src/main/java/com/amazon/dataprepper/DataPrepperExecute.java

```diff
@@ -30,7 +34,17 @@ public static void main(String[] args) {
         }
         final DataPrepper dataPrepper = DataPrepper.getInstance();
         if (args.length > 0) {
-            dataPrepper.execute(args[0]);
+            String configurationFileLocation = args[0];
+            if (args[0].endsWith(".conf")) {
+                LogstashConfigConverter logstashConfigConverter = new LogstashConfigConverter();
+                try {
+                    configurationFileLocation = logstashConfigConverter.convertLogstashConfigurationToPipeline(
+                            args[0], String.valueOf(Paths.get("data-prepper-core/build/libs/")));
```

Comment:
This is the path at compile-time. But, when a user is running Data Prepper, this path will not exist. I think just using the current working directory is sufficient for now. It will need to be revisited along with implementing #305 .

Reply:
This is in reference to the string `"data-prepper-core/build/libs/"`.

Reply:
Got it, just thinking where the .yaml file should be saved.
I think we can remove the 2nd parameter and save it in same directory as .conf file.